prow/plugins/plugins.go

```diff
@@ -363,10 +363,21 @@ type Cat struct {
 	KeyPath string `json:"key_path,omitempty"`
 }
 
+// Label is the config for the label plugin.
 type Label struct {
+	// Repos is either of the form org/repos or just org.
+	// The AdditionalLabels and Prefixes values are applicable
+	// to these repos.
+	Repos []string `json:"repos,omitempty"`
 	// AdditionalLabels is a set of additional labels enabled for use
-	// on top of the existing "kind/*", "priority/*", and "area/*" labels.
-	AdditionalLabels []string `json:"additional_labels"`
+	// on top of the existing "kind/*", "priority/*", "area/*"
+	// and "triage/*" labels.
+	// Labels can be used with `/[remove-]label <additionalLabel>` commands.
+	AdditionalLabels []string `json:"additional_labels,omitempty"`
```

Comment:
The UX here is a bit weird -- you can provide prefixes and use them with any target, or additional labels when you want to provide only specific targets? Why not some type of prefix -> target mapping? Leave targets empty or use a sentinel to approve all?